doc/tutorial/basic/tutorial.rst

```diff
@@ -33,13 +33,13 @@ We can separate learning problems in a few large categories:
       <https://en.wikipedia.org/wiki/Classification_in_machine_learning>`_:
       samples belong to two or more classes and we
       want to learn from already labeled data how to predict the class
-      of unlabeled data. An example of classification problem would
-      be the handwritten digit recognition example, in which the aim is
+      of unlabeled data. An example of a classification problem would
+      be handwritten digit recognition, in which the aim is
       to assign each input vector to one of a finite number of discrete
       categories.  Another way to think of classification is as a discrete
-      (as opposed to continuous) form of supervised learning where one has a
-      limited number of categories and for each of the n samples provided,
-      one is to try to label them with the correct category or class.
+      (as opposed to continuous) form of supervised learning where there's
+      a limited number of categories and we try to fit some data set to one
+      of those categories.
```

Comment:
I kind of like the previous formulation better in this case i.e. something "label samples with the correct category or class". IMO "fit some data set to one of those categories" feels more vague. Better suggestions welcome!

Reply:
I'll restore the old language. Thanks.